ocs_ci/deployment/deployment.py

```diff
@@ -216,6 +225,41 @@ def deploy_ocs(self):
                 f"MDS deployment Failed! Please check logs!"
             )
 
+        if config.RUN['cli_params']['monitoring']:
```

Comment:
As I said in comment above, check this option: 
https://github.com/red-hat-storage/ocs-ci/blob/c646ab1f9a87f6af02a74d971c4d08a1ddaecf53/ocs_ci/deployment/deployment.py#L121

And I think that all the monitoring related stuff should be moved to some separate method.